docs/standard/microservices-architecture/container-docker-introduction/docker-defined.md

```diff
@@ -39,21 +39,20 @@ Figure 2-3 shows a comparison between VMs and Docker containers.
+  Virtual machines include the application, the required libraries or binaries, and a full guest operating system. Full virtualization requires more resources than containerization. Containers include the application and all its dependencies. However, containers share the OS kernel with other containers. Containers run as isolated processes in user space on the host operating system. (Except in Hyper-V containers, where each container runs inside of a special virtual machine per container.)
   ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 **Figure 2-3**. Comparison of traditional virtual machines to Docker containers
 
-Because containers require far fewer resources (for example, they do not need a full OS), they are easy to deploy and they start fast. This allows you to have higher density, meaning that it allows you to run more services on the same hardware unit, thereby reducing costs.
+Because containers require far fewer resources (for example, they do not need a full OS), they start fast and are easy to deploy. Low resource usage allows higher density. You can run more services on the same hardware unit and reduce costs.
 
-As a side effect of running on the same kernel, you get less isolation than VMs.
+Running on the same kernel resultes in less isolation than VMs provide.
 
 The main goal of an image is that it makes the environment (dependencies) the same across different deployments. This means that you can debug it on your machine and then deploy it to another machine with the same environment guaranteed.
 
 A container image is a way to package an app or service and deploy it in a reliable and reproducible way. You could say that Docker is not only a technology, but also a philosophy and a process.
 
-When using Docker, you will not hear developers say, “It works on my machine, why not in production?” They can simply say, “It runs on Docker,” because the packaged Docker application can be executed on any supported Docker environment, and it will run the way it was intended to on all deployment targets (Dev, QA, staging, production, etc.).
-
+Docker developers don't say, "It works on my machine, why not in production?" They say, "It runs on Docker". Docker packaged apps can be executed on any supported Docker environment. Docker packaged apps run consistently on all deployment targets (Dev, QA, staging, production).
```

Comment:
Period inside the quote ...

> They say, "It runs on Docker."

Hyphenate "Docker packaged" :point_right: Docker-packaged

:point_up: That occurs in two sentences here.

Optional for the list that perhaps make it flow a bit better ...

> Docker packaged apps run consistently on all deployment targets: Dev, QA, staging, and production.